prijateli_tree/app/utils/games.py

```diff
@@ -0,0 +1,72 @@
+"""Utility functions for specific game types."""
+
+from prijateli_tree.app.utils.constants import (
+    NETWORK_TYPE_INTEGRATED,
+    NETWORK_TYPE_SEGREGATED,
+    NETWORK_TYPE_SELF_SELECTED,
+)
+
+
+class Game:
+    def __init__(
```

Comment:
There's no need to create a `Game` class with actual values outside of its neighbors.